scripts/codegen.js

```diff
@@ -15,6 +15,10 @@ telescope({
   ],
   outPath: outPath,
   options: {
+    aminoEncoding: {
+      enabled: true,
+      useRecursiveV2encoding: true,
+    },
```

Comment:
This option also appears at the end of the options in this file as `aminoEncoding: { enabled: false }` - not sure if the later one is having any impact.

Reply:
Oh, indeed, that is a bug. Thank you!